### ISSUE TITLE
csserver: FATAL ERROR (shutting down): W_LoadWadFile: couldn't load gfx.wad

### ISSUE THREAD

**Issue by kylephp** [akshmakov/LinuxGSM-docker#1]
Hello,
I tried to follow the instructions to setup a new csserver, everything was good but the server could not be started. Here is the debug information:

> Console initialized.
Using breakpad crash handler
Setting breakpad minidump AppID = 10
Forcing breakpad minidump interfaces to load
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
FATAL ERROR (shutting down): W_LoadWadFile: couldn't load gfx.wad
email debug.log to linux@valvesoftware.com
Fri Jul 28 06:32:26 UTC 2017: Server restart in 10 seconds

Any idea?

**Comment by akshmakov** [comment]
I replicated the problem, but it does not appear to be related to anything with the docker environment, or even LinuxGSM, but rather a SteamCMD issue. It doesn't appear to be downloading all the asset files. 


Some folks online said to try from scratch a few times, to retry the download.

On a different project I had succesfuly downloaded app assets for a game server with no issues, so I don't suspect the base image , but I will run  a different base image and see if it helps.


GameServerManagers/LinuxGSM#1450
ValveSoftware/halflife#928 (old, but maybe similar root cause)


Literally from LinuxGSM logs (from steamcmd)

```
Information! Goldsource servers commonly fail to download all the server files r
equired. Validating a few of times may reduce the chance of this issue.
```

**Comment by kylephp** [comment]
@akshmakov yes, I agree that was SteamCMD issue. I managed to fix it by manually downloading hlds for linux. I have successfully started the csserver but when I was connecting to it, I got this issue: the port 27015 closed after client connected
![screenshot from 2017-08-01 06-59-55](https://user-images.githubusercontent.com/7766055/28803613-0ff73d74-7688-11e7-92cd-1ffe7c207db9.png)
I know it's not your project issue but just wonder you have any idea?

**Comment by ciro25** [comment]
Download this file below:
Download Decals.wad here :  https://mega.nz/#!d0VGTCZL!m8LsUS5vnjUneexPxwQFxk-PIDoT5x5Nq3N1uxHN_dE
Copy it into your cstrike folder.
Restart server and your problem is solved.
Thank You

**Comment by Robirt55** [comment]
@kylephp Did you forward the ports on your server, and have you tested with Telnet? 

Could you post instructions on manually downloading the hlds files?

**Comment by dnhuan** [comment]
Guys, I have found the solution, 
I downloaded the game (Client) through Desktop Steam on my 2nd PC and copy the original gfx.wad file to the "serverfiles" folder on the server (Which is missing that file). Here is the gfx.wad file I copied (https://drive.google.com/file/d/1YvSFdOqsTdBVVQ3HB-kWNDw9GYrhDqaQ/view?usp=sharing)

**Comment by ladislavsulc** [comment]
I have the same problem. STILL. Will keep you updated on possible solutions.

**Comment by shlomo1234** [comment]
Hello,
gfx.wad should be in /valve/gfx.wad
The problem is that this file is missing from this folder, please try redownloading the CS 1.6 files from steam and check if this file is there now. Enjoy.
